domehunter/enumerations.py

```diff
@@ -0,0 +1,101 @@
+from enum import Flag
+
+
+class Direction(Flag):
+    CW = True
+    CCW = False
+    none = None
+
+
+class LED_light(Flag):
```

Comment:
`LED_light` is a class so PEP8 suggests camel case with underscores, i.e. `LED_Light`.  For readability I'd call it `LED_Lights`.